src/commands/game_commands/leaderboard.ts

```diff
@@ -35,32 +47,32 @@ export default class LeaderboardCommand implements BaseCommand {
     help = {
         name: "leaderboard",
         description: "View the KMQ leaderboard.",
-        usage: ",leaderboard\n,leaderboard [server | game] {page_number}\n,leaderboard page {page_number}\n,leaderboard [enroll | unenroll]",
+        usage: ",leaderboard\n,leaderboard {server | game} {page_number} {daily | weekly | monthly}\n,leaderboard [enroll | unenroll]",
         examples: [
             {
                 example: "`,leaderboard`",
                 explanation: "Show the global leaderboard",
             },
             {
-                example: "`,leaderboard page 3`",
+                example: "`,leaderboard 3`",
                 explanation: "Shows the 3rd page of the global leaderboard",
             },
             {
                 example: "`,leaderboard server`",
                 explanation: "Shows the server-wide leaderboard",
             },
             {
-                example: "`,leaderboard server 3`",
+                example: "`,leaderboard 3 server`",
                 explanation: "Shows the 3rd page of the server-wide leaderboard",
             },
-            {
-                example: "`,leaderboard game`",
-                explanation: "Shows the leaderboard of players with points in the current game",
-            },
             {
                 example: "`,leaderboard game 2`",
                 explanation: "Shows the 2nd page of the players with points in the current game",
             },
+            {
```

Comment:
Inconsistency on where the page parameter goes is not user friendly
`,leaderboard 3 server`
`,leaderboard weekly 4`

Reply:
I mainly added it to show order doesn't matter, would you prefer having the page number at the end for both of them?